decorator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'nameable'
+
+class Decorator
+  include Nameable
+
```

Comment:
- [ ] Kindly consider adding getters and setters for `nameable`, you can do this by adding it to `attr_accessor` 
e.g `attr_accessor :nameable`